rules/mapping.go

```diff
@@ -71,6 +72,8 @@ func newMappingRuleSnapshot(
 		r.TagFilters,
 		policies,
 		filter,
+		r.LastUpdatedAt,
+		r.LastUpdatedBy,
 	), nil
 }
 
```

Comment:
suggestion(take/leave): merge `newMappingRuleSnapshotFromFields` into `newMappingRuleSnapshot`.

Reply:
I'm gonna leave this like this because if there's ever a need for a second constructor someone will have to make this function again. That's the reason it was there in the first place.